isisd/isis_spf.c

```diff
@@ -904,6 +657,9 @@ static int isis_spf_process_lsp(struct isis_spftree *spftree,
 			&lsp->tlvs->oldstyle_ip_reach_ext};
 
 		for (unsigned int i = 0; i < array_size(reachs); i++) {
+			if (fabricd)
```

Comment:
I think it should be possible to move this check to the `if` statement above to get rid of one of the SA warnings. Example:
```c
    if (!pseudo_lsp && spftree->family == AF_INET
        && spftree->mtid == ISIS_MT_IPV4_UNICAST
        && !fabricd) {
```